EasyPost.Net/Event.cs

```diff
@@ -18,9 +18,9 @@ public class Event : Resource
         public DateTime? updated_at { get; set; }
 
         /// <summary>
-        ///     Resend the last Event for a specific EasyPost object instance.
+        ///     Resend the last Event for a specific EasyPost.Net object instance.
```

Comment:
These comment update needs to be removed

Reply:
Resolved in [3b9751f](https://github.com/EasyPost/easypost-csharp/pull/209/commits/3b9751ffe837360356709749f90f25a116898726)